### PR TITLE
Update README due to Users changes

### PR DESCRIPTION
**Pull request by alexcameron89**
Updates based on #134

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ $ bin/setup
 $ rails s
 ```
 
-Then navigate to `http://localhost:3000` in your browser to view the app. Two user accounts will be available for logging in: **User** and **Admin**. Use one of the logins to access the app:
- * User
-   * Email: `user@example.com`
```
Comment (atzorvas) [inline README.md:50]: While I like that you've updated documentation for admin accounts, it seems that you've removed from the documentation the simple user account :$

Comment (alexcameron89) [inline README.md:50]: Right, thank you, I thought that was removed. I'll update it.

**Comment by atzorvas** [comment]
:+1: 

@alexcameron89 thanks for fixing this!

**Comment by alexcameron89** [comment]
Thanks! Once I get more time to wrap my head around the project, I'm hoping to help on some of the issues.

**Pull request merged**